zipkin-collector-service/src/main/scala/com/twitter/zipkin/collector/builder/CollectorServiceBuilder.scala

```diff
@@ -39,6 +40,7 @@ import com.twitter.zipkin.thriftscala
  */
 case class CollectorServiceBuilder[T](
   storeBuilder: Builder[Store],
+  receiver: Option[(Seq[thriftscala.Span] => Future[Unit]) => SpanReceiver] = None,
```

Comment:
I don't suppose theres something we could do to simplify this?  Do you think it would be worth abstracting this to some kind of type `type SpanReceiver = A => B => C`.